gw_eccentricity/eccDefinition.py

```diff
@@ -584,6 +584,17 @@ def process_data_dict(self,
         amp22_merger = newDataDict["amplm"][(2, 2)][merger_idx]
         phase22 = newDataDict["phaselm"][(2, 2)]
         phase22_merger = phase22[merger_idx]
+        # check if phase22 is increasing. phase22 at merger should be greater
+        # than the phase22 at the start of the waveform
+        if phase22_merger < phase22[0]:
+            raise Exception(
+                f"Phase22 = {phase22_merger} at the merger is < "
```

Comment:
Phase22 --> phase22, to be consistent with our own documentation:
https://github.com/vijayvarma392/gw_eccentricity/blob/b96e69dd373dca54fe18ed71676aabda863ff753/gw_eccentricity/eccDefinition.py#L38

Reply:
Done